spec/mongoid/query_cache_spec.rb

```diff
@@ -123,6 +123,27 @@
         end
       end
     end
+
+    context "when cache is enabled and second batch of items is requested" do
+
+      before do
+        Mongoid::QueryCache.enabled = true
```

Comment:
I think `around` will be better here, because you can't be sure what's the initial value.

```ruby
around(:each) do |example|
  query_cache_enabled = Mongoid::QueryCache.enabled
  Mongoid::QueryCache.enabled = false
  example.run
  Mongoid::QueryCache.enabled = query_cache_enabled
end
```